assets/pages/showcase/components/otherProducts.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react';
+
+import ProductPageContentBlock from 'components/productPage/productPageContentBlock/productPageContentBlock';
+import ProductPageTextBlock from 'components/productPage/productPageTextBlock/productPageTextBlock';
+
+import OtherProduct from './otherProduct';
+
+export default function OtherProducts() {
+  return (
+    <ProductPageContentBlock modifierClasses={['other-products']}>
+      <ProductPageTextBlock title="Other ways you can support us">
+        <OtherProduct
+          title="The Guardian Patrons"
+          description="Support from our Patrons is crucial to ensure that generations to come will be able to enjoy The Guardian"
+          destination="https://patrons.theguardian.com"
```

Comment:
There are some helpers in `externalLinks.js` that could help with building this URL and the membership one below.

Reply:
done